package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "chokidar '(client|server|test)/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
I found a good solution! Using `nodemon` instead of `chokidar` solves tons of problems💃